fontbe/src/fvar.rs

```diff
@@ -33,6 +37,15 @@ fn generate_fvar(static_metadata: &StaticMetadata) -> Option<Fvar> {
     // Reuse an existing name record if possible.
     let reverse_names = static_metadata.reverse_names();
 
+    // If a single postscript name is present, we must provide one or explicitly
+    // indicate absence for every instance. fontations only expects None when NO
+    // instances have one.
+    // https://github.com/googlefonts/fontations/blob/b4136692/write-fonts/src/tables/fvar.rs#L13-L21
+    let postscript_names = static_metadata
```

Comment:
nit: I would expect `postscript_names` to be the actual names, vs something like `has_postscript_names` that would capture whether there are any.

Reply:
That makes sense and I prefer how it reads too, updated now